src/layout/Likert/RepeatingGroupsLikertContainer.test.tsx

```diff
@@ -327,8 +321,7 @@ describe('RepeatingGroupsLikertContainer', () => {
         mobileView: true,
       });
 
-      // Should have 2 alerts per validation error (one on the radio button and one below with the error message)
-      expect(screen.getAllByRole('alert')).toHaveLength(2 + 2);
```

Comment:
Ser det her tidligere har blitt testet at det er 4 `Alerts`, men nå er det kun 2. Er dette gjort med hensikt? Eller er dette relatert til denne kommentaren? 

> Et annet problem er at Radioknapper ikke har muligheten alene til å vise feil/error. Så dette blei ikke løst. Det bør derfor kanskje opprettes en bug i digdir design system om dette? Vi bruker denne funksjonaliteten i Likert med validering.

Reply:
Ja godt observert. Her gikk det kanskje litt fort i svingen. Dette har med mobilvisning av Likert å gjøre. Feilmeldingen for radiogrupper vises utenfor komponenten. Det gjør at man får to alerts i gammel kode. Her ser man Legacy html'en og komponent:
![image](https://github.com/Altinn/app-frontend-react/assets/43168787/f7c4b0dd-5013-4e29-a9e4-b9937765be3e)
![image](https://github.com/Altinn/app-frontend-react/assets/43168787/b8d7ac02-9a21-4904-bb61-fe4f390a7acf)


I ny Radio er role 'alert' fjernet inni radiogruppen og heller brukt aria-live attribut:
![image](https://github.com/Altinn/app-frontend-react/assets/43168787/dd2ba1de-ce51-46f3-8731-2f9da9967334)
![image](https://github.com/Altinn/app-frontend-react/assets/43168787/b6a8b248-6758-47e2-b6a1-97ecd357ebb1)


Jeg foreslår egentlig her at vi burde flytte feilmeldingen inni RadioGroup komponenten og fjerne den utenfor. Siden nå er det bare en tom feilmelding inni (det var også dette før). Når jeg snakker om utenfor mener jeg den som er rendret i GenericComponent.

Reply:
Den er god! Vi lar det være som det er nå. Da kommer du videre med issuet, og vi endrer ingen oppførsel. Så oppretter vi heller et issue om å få feilmeldingene inn i `Radio.Group` på sikt dersom vi ønsker å gå den veien.